src/integTest/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoaderIntegrationTest.groovy

```diff
@@ -182,4 +182,35 @@ class DockerThreadContextClassLoaderIntegrationTest extends ProjectBuilderIntegr
         instance
         instance.links.links.size() == 0
     }
+
+    def "Can create class of type Bind"() {
+
+        when:
+        def instance = null
+
+        threadContextClassLoader.withClasspath(project.configurations.dockerJava.files, dockerClientConfiguration) {
+            instance = createBind("/my/path", "/my/other/path")
+        }
+
+        then:
+        noExceptionThrown()
+        instance
```

Comment:
We should also check the fields (volume, path) of the `Bind` element here to make sure the assignment works properly.